domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Services/TransferAgreementProposalCleanupServiceTests.cs

```diff
@@ -94,7 +94,7 @@ public async Task ShouldCreateActivityLogEntry_WhenDeleting()
         dbContext.TransferAgreementProposals.Add(oldInvitation);
         await dbContext.SaveChangesAsync();
 
-        var invitations = await dbContext.RepeatedlyQueryUntilCountIsMet<TransferAgreementProposal>(0);
+        var invitations = await dbContext.RepeatedlyQueryUntilCountIsMet<TransferAgreementProposal>(0, TimeSpan.FromSeconds(30));
```

Comment:
Why this change?

Reply:
Because the test is flaky :( Needed to give it more time to complete. Not sure why it's flaky. Maybe it's something to do with us spinning so many docker images up